persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/impl/table/SybaseTableManager.java

```diff
@@ -65,4 +72,24 @@ protected String initUpsertRowSql() {
                config.segmentColumnName());
       }
    }
+
+   @Override
+   protected boolean indexExists(String indexName, Connection conn) throws PersistenceException {
+      ResultSet rs = null;
+      try {
+         DatabaseMetaData meta = conn.getMetaData();
+         rs = meta.getIndexInfo(null, tableName.getSchema(), tableName.getName(), false, false);
+
+         while (rs.next()) {
+            if (rs.getString("INDEX_NAME") != null && indexName.equalsIgnoreCase(rs.getString("INDEX_NAME").replaceAll("\"", ""))) {
```

Comment:
If we do `String index = rs.getString("INDEX_NAME"); if (index != null ...)` we only make a single call to the JDBC driver, also from the `ResultSet` docs:

```quote
For maximum portability, result set columns within each row should be read in left-to-right order, and each column should be read only once.
```
https://docs.oracle.com/javase/8/docs/api/java/sql/ResultSet.html

Reply:
@ryanemerson Good catch, it was updated, thanks